test/issue-1384.js

```diff
@@ -0,0 +1,15 @@
+import {expect} from '../index.js';
+
+describe('Fix #1384: Empty keys support', function () {
+  it('should accept empty keys when the object is empty', function () {
```

Comment:
these should be added to the existing `expect` tests, either as part of the existing test case or just move these two new ones and reword them